optuna/visualization/_pareto_front.py

```diff
@@ -22,6 +26,15 @@
 _logger = optuna.logging.get_logger(__name__)
 
 
+class _ParetoFrontInfo(NamedTuple):
+    n_targets: int
+    target_names: List[str]
```

Comment:
[question]

I found `List` and `Sequence` both appear in this file. Do you have any policy to distinguish uses of them?

Reply:
I just had a vague policy that I use `List` if the container is created in this code and `Sequence` if the container is given by users. But now this distinction seems unnecessary. I will change `List` to `Sequence`.

Reply:
Thank you.

> Some distinctions have been existing before this change

It's okay to leave them. After this visualization refactoring was done, we'll another PR to cleanup the signatures of utilities in visualization modules.